recipes-core/acrn/acrn-hypervisor.bb

```diff
@@ -15,6 +15,13 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
 DEPENDS += "python3-kconfiglib-native"
 DEPENDS += "${@'gnu-efi' if d.getVar('ACRN_FIRMWARE') == 'uefi' else ''}"
 
+do_configure() {
+	cat <<-EOF >> ${S}/hypervisor/arch/x86/configs/${ACRN_BOARD}.config
+CONFIG_$(echo ${ACRN_SCENARIO} | tr '[:lower:]' '[:upper:]')=y
```

Comment:
create a blank file should be sufficient.
CONFIG_SCENARIO are now replaced CONFIG_${SCENARIO}
https://github.com/projectacrn/acrn-hypervisor/commit/dba0591f72d0f070b87e90ed83c83c4c505b8cdf

Reply:
CONFIG_${SCENARIO}, is internal variable. Anyway we are adding in *.config, so I see no issue.